### PR TITLE
Colors

### PR DESCRIPTION
**Pull request by kjgross**
surface nba colors to front end.

### CODE REVIEW

**Comment by blakeparkinson** [comment]
whoa you are on :fire:

File: models/players.js [hunk 1]
```diff
@@ -15,13 +15,24 @@ var parseString = require('xml2js').parseString;
 var players = [];
 var encryption = require('../encryption.js');
 var shortId = require('shortid');
+var team_colors_nba = require('../lists/team_colors/team_colors_nba.js')
 
 
-var returnPlayers = function (players, rb_team_id, res, league){
+var intreturnPlayers = function(players, rb_team_id, res, league, second_callback){
```
Comment (blakeparkinson) [inline models/players.js]: what does this function name mean? Why isnt it camelcase? Also, why is it needed? It doesnt really do anything,

File: models/players.js [hunk 2]
```diff
@@ -38,6 +49,26 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var fetchTeamColors = function (league, team_name){
+  var colorsObject = _.first(team_colors_nba.team_colors);
+  if (league == "nba"){
```
Comment (blakeparkinson) [inline models/players.js]: switch statement will be nicer

File: models/players.js [hunk 3]
```diff
@@ -38,6 +49,26 @@ var returnPlayers = function (players, rb_team_id, res, league){
...
+    for (i=0;i<colorsObject.teams.length;i++){
+      if (colorsObject.teams[i].team_name == team_name){
+        var colors = [];
```
Comment (blakeparkinson) [inline models/players.js]: save a line of code by defining colors at the top of the function. Also its a little more readable when the variables are defined at the start

File: models/players.js [hunk 4]
```diff
@@ -38,6 +49,26 @@ var returnPlayers = function (players, rb_team_id, res, league){
...
+        colors.primary_hex = colorsObject.teams[i].primary_hex;
+        colors.secondary_hex = colorsObject.teams[i].secondary_hex;
+        return colors;
+      }
+    }
+  }
+  else{ 
+    var colors = [];
+    colors.primary_hex = "#000080";
```
Comment (blakeparkinson) [inline models/players.js]: default RB text is `#333333`

File: models/players.js [hunk 5]
```diff
@@ -38,6 +49,26 @@ var returnPlayers = function (players, rb_team_id, res, league){
...
+    colors.secondary_hex = "#FF0000"
```
Comment (blakeparkinson) [inline models/players.js]: isnt this red? Should this be white FFFFFF

File: models/players.js (hunk 1 above)
Comment (kjgross) [inline models/players.js]: It's for "intermediate".. idk.

Reason I have it is I was running into async problems. Any ideas for not using two callbacks in a row?

Comment (blakeparkinson) [inline models/players.js]: ah i see. Okay then this is fine, Good work !

**Pull request merged**